index.js

```diff
@@ -11,29 +11,134 @@ const CLIENT_INFO = `${sdkPkg.name}/${sdkPkg.version}`;
 const ENV_INFO = `${seleniumPkg.name}/${seleniumPkg.version}`;
 const utils = require('@percy/sdk-utils');
 const { DriverMetadata } = require('./driverMetadata');
+const log = utils.logger('selenium-webdriver');
+
+const getWidthsForMultiDOM = (userPassedWidths, eligibleWidths) => {
+  // Deep copy of eligible mobile widths
+  let allWidths = [];
+  if (eligibleWidths?.mobile?.length !== 0) {
+    allWidths = allWidths.concat(eligibleWidths?.mobile);
+  }
+  if (userPassedWidths.length !== 0) {
+    allWidths = allWidths.concat(userPassedWidths);
+  } else {
+    allWidths = allWidths.concat(eligibleWidths.config);
```

Comment:
can eligibleWidths not be null here?

Reply:
it can be null for older cli. but not for newer cli's. and this flow is only triggered when responsiveSnapshotCapture is true which is supported after 1.29.5-beta.0.

Reply:
got it.